packages/mdc-slider/README.md

```diff
@@ -0,0 +1,274 @@
+<!--docs:
+title: "Sliders"
+layout: detail
+section: components
+iconId: slider
+path: /catalog/input-controls/sliders/
+-->
+
+# Slider
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/slider.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/sliders.png" width="400" alt="Select screenshot">
+  </a>
+</div>-->
+
+> **Status**:
+> - [x] Continuous Sliders
+> - [ ] Discrete Sliders
+
+MDC Slider provides an implementation of the Material Design slider component. It is modeled after
+the browser's `<input type="range">` element. Sliders are fully RTL-aware, and conforms to the
+WAI-ARIA [slider authoring practices](https://www.w3.org/TR/wai-aria-practices-1.1/#slider).
+
+Note that **vertical sliders and range (multi-thumb) sliders are not supported, due to their absence
+from the material design spec**.
+
+Also note that we have taken certain deviations from the UX within the spec, so there may be some
+treatments which deviate from the mocks. These deviations arose out of design feedback from seeing
+sliders used on the web, and thus have been endorsed by the Material Design team.
+
+## Design and API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/sliders.html">Material Design guidelines: Sliders</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm i --save @material/slider
+```
+
+## Usage
+
+```html
+<div class="mdc-slider" tabindex="0" role="slider"
+     aria-valuemin="0" aria-valuemax="100" aria-valuenow="0"
+     aria-label="Select Value">
+  <div class="mdc-slider__track-container">
+    <div class="mdc-slider__track"></div>
+  </div>
+  <div class="mdc-slider__thumb-container">
+    <svg class="mdc-slider__thumb" width="21" height="21">
+      <circle cx="10.5" cy="10.5" r="7.875"></circle>
+    </svg>
+    <div class="mdc-slider__focus-ring"></div>
+  </div>
+</div>
+```
+
+Then in JS
+
+```js
+import {MDCSlider} from '@material/slider';
+
+const slider = new MDCSlider(document.querySelector('.mdc-slider'));
+slider.listen('MDCSlider:change', () => console.log(`Value changed to ${slider.value}`));
+```
+
+You can also include MDCSlider via its UMD version located at `dist/mdc.slider[.min].js`
+
+```js
+// CommonJS
+const {MDCSlider} = require('@material/slider/dist/mdc.slider');
```

Comment:
It actually doesn't, since in `package.json` we specify the package name as `@material/slider`